components/aggregation/website/CountrySummary.js

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from 'react'
+import { Flex, Box, Heading, Text, Link } from 'ooni-components'
+import styled from 'styled-components'
+import { IoCloseCircled } from 'react-icons/lib/io'
+import { FaBarChart, FaExclamationCircle } from 'react-icons/lib/fa'
+import { MdCheckCircle } from 'react-icons/lib/md'
+import NLink from 'next/link'
+import { useRouter } from 'next/router'
+
+import {
+  colorNormal,
+  colorAnomaly,
+  colorConfirmed,
+} from '../../colors'
+import BoxChart from './BoxChart'
+import { paramsToQuery } from '../../../components/aggregation/website/queryUtils'
+
+const ICON_SIZE = 30
+
+const BorderedBox = styled(Box)`
+  border: 1px solid ${props => props.theme.colors.gray4};
+`
+
+const CountrySummary = ({ data }) => {
+  const {
+    anomaly_count,
+    confirmed_count,
+    failure_count,
+    measurement_count,
+    probe_cc
+  } = data
+
+  const ok_count = measurement_count - (anomaly_count + confirmed_count + failure_count)
+
+  let outcome = {
+    color: colorNormal,
+    subtext: 'not blocked',
+    percent: Number(ok_count / measurement_count * 100).toFixed(0),
+    icon: <MdCheckCircle size={ICON_SIZE} />
+  }
+
+  if (anomaly_count > ok_count) {
+    outcome = {
+      color: colorAnomaly,
+      subtext: 'high anomaly count',
+      percent: Number(anomaly_count / measurement_count * 100).toFixed(0),
+      icon: <FaExclamationCircle size={ICON_SIZE} />
+    }
+  }
+
+  if (confirmed_count > ok_count) {
+    outcome = {
+      color: colorConfirmed,
+      subtext: 'confirmed blocked',
+      percent: Number(confirmed_count / measurement_count * 100).toFixed(0),
+      icon: <IoCloseCircled size={ICON_SIZE} />
+    }
+  }
```

Comment:
The percentage for a country should be:

Total of anomalous measurement (anomaly_count + confirmed_count) / total_measurement_count

We should display in the list first countries which have a confirmed count that is non-zero, then it should be sorted by the the percentages.

It would be useful to have input from @FedericoCeratto on this as well.